### ISSUE TITLE
NIPS '17 | Train longer, generalize better: closing the generalization gap in large batch training of neural networks

### ISSUE THREAD

**Issue by vycezhong** [vycezhong/read-papers#113]
https://arxiv.org/pdf/1705.08741.pdf

很早看的，忘标了

**Comment by vycezhong** [comment]
现在来看，这篇文章的观点有点过时了，至少没有被广泛采纳

____

这篇文章前面介绍了一堆我看不懂的东西，把SGD比喻成在loss曲面上做random walk啦，反正得到一个结论：在刚开始的训练中，||w_t - w_0||的L2随时间log增长。

公式化表达是这样
![image](https://user-images.githubusercontent.com/25879526/92307689-b8bc4580-efca-11ea-8e52-2d903324b906.png)

实验也验证了这一点
![image](https://user-images.githubusercontent.com/25879526/92307744-1d77a000-efcb-11ea-9b5f-915c84592dd4.png)

但上图表面，不同的batch size，1) 看上去增长的速度不太一样 2)更新的次数也不一样

然后他们做了两点努力：
1. square root LR scaling
2. ghost batch normalization (GBN)，这名字听起来很有b格，但其实就是只用一部分batch来算batch normalization....

然后又说，updates不够，updates数目要和之前一样。

综合以上几点，他们终于消除了generalization gap.....

____

BTW，现在没人这么做了……只剩下一些理论上的意义.
